clusters/openstack/icehouse/profiles/cloud-controller.example.org.pan

```diff
@@ -0,0 +1,17 @@
+###############################################################################
```

Comment:
remove this header or use pan annotations like http://quattor-pan.readthedocs.org/en/latest/pan-book/pan-book.html#annotations